From 377b650fb1162d989f584cce69cbc65fc10d1579 Mon Sep 17 00:00:00 2001
From: FabioLolix <fabio.loli@disroot.org>
Date: Fri, 25 Oct 2024 13:42:11 +0200
Subject: [PATCH] remove error for not recognized package_type

---
 cmake/packaging/setup.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/packaging/setup.cmake b/cmake/packaging/setup.cmake
index 43cf84c..de6e751 100644
--- a/cmake/packaging/setup.cmake
+++ b/cmake/packaging/setup.cmake
@@ -42,7 +42,7 @@ if (EXISTS "/etc/debian_version")
 elseif(EXISTS "/etc/portage")
     set(PACKAGE_TYPE "ebuild")
 else()
-    message(FATAL_ERROR "Unable to detect package type for this system")
+    return()
 endif()
 
 include(${CMAKE_CURRENT_LIST_DIR}/generators/${PACKAGE_TYPE}.cmake)
-- 
2.47.0

